src/pages/Login.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+export function Login() {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```

Comment:
Type of these states are infered from inistial value, so generics are not necessary.